sdk/core/azure-core/src/test/java/com/azure/android/core/internal/util/serializer/ByteArraySerializerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.azure.android.core.internal.util.serializer;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ByteArraySerializerTest {
+
+    @Test
+    public void test_getModule() {
+        SimpleModule module = ByteArraySerializer.getModule();
+
+        assertNotNull(module);
+    }
+
+    @Test
+    public void test_serialize() throws IOException {
+        Byte[] byteObjects = new Byte[] { 84, 101, 115, 116, 32, 115, 116, 114, 105, 110, 103 };
```

Comment:
Does this need to be the object type of `byte`?

Reply:
Yes. The [`serialize`](https://github.com/Azure/azure-sdk-for-android/blob/f63b20c9b079f5e2f3157d557db0b7c179fb9001/sdk/core/azure-core/src/main/java/com/azure/android/core/internal/util/serializer/ByteArraySerializer.java#L31) method we override in `ByteArraySerializer` expects a `Byte[]` object.